server/repository/src/db_diesel/requisition/requisition.rs

```diff
@@ -242,6 +243,25 @@ fn create_filtered_query(
                 query = query.filter(requisition::id.nullable().ne_all(requisition_ids))
             }
         }
+
+        if let Some(has_outstanding_lines) = has_outstanding_lines {
+            use crate::db_diesel::requisition_line_row::requisition_line;
+
+            let requisition_ids = requisition_line::table
+                .select(requisition_line::requisition_id)
+                .group_by(requisition_line::requisition_id)
+                .having(
```

Comment:
@mark-prins was alright using requested vs supply instead of what is already supplied in Outbound Shipments